dev/Code/CryEngine/RenderDll/Common/Renderer.cpp

```diff
@@ -4026,6 +4026,15 @@ void CRenderer::RenderTextMessages(CTextMessages& messages)
             }
 
             ProjectToScreen(vPos.x, vPos.y, vPos.z, &sx, &sy, &sz);
+			
+            if (!b800x600)
+            {
+                // ProjectToScreen() returns virtual screen values in range [0-100], while the Draw2dTextWithDepth() method expects screen coords.
+                // Correcting sx, sy values if not in virtual screen mode (sz is depth in range [0-1], and does not need to be altered).
+                SViewport& vp = m_pRT->IsRenderThread() ? m_NewViewport : m_MainRTViewport;
```

Comment:
This line seems redundant no?

Reply:
Just checking the source fixes now... seems odd...

Reply:
Hmmm yes, good catch, I'm not sure how that got in there!